client/src/wh/events.cljs

```diff
@@ -78,12 +78,12 @@
   ::toggle-job-like-success
   db/default-interceptors
   (fn [{db :db} [id action]]
-    {:db (update-in db [:wh.user.db/sub-db :wh.user.db/liked-jobs] util/toggle id)
-     :dispatch (case action
-                     :reload-recommended [:personalised-jobs/fetch-jobs-by-type :recommended 1]
-                     :reload-liked [:personalised-jobs/fetch-jobs-by-type :liked 1]
-                     :reload-dashboard [:wh.logged-in.dashboard.events/fetch-recommended-jobs]
-                     [])}))
+    (merge {:db (update-in db [:wh.user.db/sub-db :wh.user.db/liked-jobs] util/toggle id)}
```

Comment:
This is much better than what I did :sweat_smile: , as it means components don't need to remember or know about supplying an action, and if they are on a page that would like a job but not re-render , then there is no need for action to be called . Thank you for helping me out @acron0

Reply:
No prob :) The `case` clause was causing a few errors in prod so our system picked it up and I threw a fix in.